src/service/api.tsx

```diff
@@ -19,35 +19,43 @@ const api = {
   getStore() {
     const serviceStore: any = localStorage.getItem(apiService.storeKey);
     serviceStore !== null
-      ? apiService.store = JSON.parse(serviceStore)
+      ? (apiService.store = JSON.parse(serviceStore))
       : this.setStore();
     return apiService.store;
   },
 
-  setStore(data: Object = {
-    history: [],
-    favorites: [],
-    theme: 1
-  }) {
+  setStore(
+    data: Object = {
+      history: [],
+      favorites: [],
+      theme: 1
+    }
+  ) {
     localStorage.setItem(apiService.storeKey, JSON.stringify(data));
   },
 
   switchTheme() {
     const store = this.getStore();
-    store.theme = (store.theme === Theme.light) ? Theme.dark : Theme.light;
+    store.theme = store.theme === Theme.light ? Theme.dark : Theme.light;
     this.setStore(store);
     return store.theme;
   },
 
   setFavoritesId(id: number) {
     const store = this.getStore();
     const isThereAnId = store.favorites.includes(id);
-    if (isThereAnId) {
-      store.favorites.push(id)
-    };
+    if (!isThereAnId) {
```

Comment:
please, describe changes of this line in description as well

Reply:
Here an isThereAnId variable holds information either there is any id in favorites on local storage or not(boolean value). Current changes allow to correctly check data in local storage and eventually will use to  add or delete id from local storage as well as change a state of favorite icon

Reply:
maybe rename to isNewId

Reply:
already changed